### PR TITLE
Update all versions 202009.00 to 202011.00

### PR DESCRIPTION
**Pull request by sarenameas**
Update all versions in CSDK from 202009.00 to 202011.00.

By submitting this pull request, I confirm that my contribution is made under the terms of the MIT license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
+We have released version 202011.00 of the AWS IoT Device SDK for Embedded-C (C-SDK). This release includes refactored MQTT, JSON Parser, and AWS IoT Device Shadow libraries for optimized memory usage and modularity. These libraries are included in the SDK via [Git submoduling](https://git-scm.com/book/en/v2/Git-Tools-Submodules). These libraries have gone through code quality checks including verification that no function has a [GNU Complexity](https://www.gnu.org/software/complexity/manual/complexity.html) score over 8, and checks against deviations from mandatory rules in the [MISRA coding standard](https://www.misra.org.uk/MISRAHome/MISRAC2012/tabid/196/Default.aspx). Deviations from the MISRA C:2012 guidelines are documented under [MISRA Deviations](MISRA.md). These libraries have also undergone both static code analysis from [Coverity static analysis](https://scan.coverity.com/), and validation of memory safety and data structure invariance through the [CBMC automated reasoning tool](https://www.cprover.org/cbmc/).
```
Comment (sarenameas) [inline README.md:7]: This paragraph may need update. That is a separate scope from version update.

File: demos/defender/defender_demo/core_mqtt_config.h [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * AWS IoT Device SDK for Embedded C V202009.00
```
Comment (leegeth) [inline demos/defender/defender_demo/core_mqtt_config.h:2]: Is it going to be capital V or lowercase v?

Comment (sarenameas) [reply demos/defender/defender_demo/core_mqtt_config.h:2]: We are going to leave the V as it as the last release.

**Review by leegeth — APPROVED** [review]

**Review by aggarw13 — APPROVED** [review]

**Review by gkwicker — APPROVED** [review]

**Comment by Kristallfee** [comment]
The API Documentation Link in the README.md does not work

**Pull request merged**